src/plugins/screenshot_mode/common/get_set_browser_screenshot_mode.ts

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
That's a good question, it doesn't 😅 . It is intended for use by reporting which injects this function into the "preload" phase of opening a page.

Happy to capture that in a comment or consider a different implementation. WDYT?

Reply:
That makes sense, thanks! I think mentioning this in the comment would be beneficial, just in case someone like me stumbles upon this file in the future.

Reply:
++ it took me a minute to understand it too - but it's pretty cool how it works